### ISSUE TITLE
training setting issue

### ISSUE THREAD

**Issue by sun-yue2002** [1e12Leon/RemoteSAM#11]
Thank you for sharing open-source code! I was wondering whether the validation split of RRSISD and RisBench datasets is used during the training process.

**Comment by 1e12Leon** [comment]
We don't introduce any dataset's validation parts in the training data.

**Comment by sun-yue2002** [comment]
> We don't introduce any dataset's validation parts in the training data.

<img width="1214" height="515" alt="Image" src="https://github.com/user-attachments/assets/c3645aef-ea1f-4041-8204-f3dae95ab999" />

<img width="1194" height="478" alt="Image" src="https://github.com/user-attachments/assets/607ac6c3-3e41-42de-8ad2-6b0930804198" />

Thank you for your reply. Additionally, I would like to know whether the results in these two tables are based on the validation sets or the test sets. We noticed that in the original RMSIN paper, the evaluations are conducted on both the validation and test sets, but your results seem to differ from both.

**Comment by 1e12Leon** [comment]
These are the results of the test set. The RMSIN result is what we reproduced, so it is slightly higher than in the original paper.

**Comment by sun-yue2002** [comment]
<img width="761" height="568" alt="Image" src="https://github.com/user-attachments/assets/6be429d4-ec4a-4fa1-8008-9b3592afe309" />

I noticed that the split of rrsisd is set to 'val' in this .sh file. Is this just an example, if I may ask? 

> These are the results of the test set. The RMSIN result is what we reproduced, so it is slightly higher than in the original paper.这些是测试集的结果。RMSIN结果是我们复制的，所以它比原始论文略高。

**Comment by 1e12Leon** [comment]
Yep, it is just an example.

**Comment by sun-yue2002** [comment]
> Yep, it is just an example.

Got that! Thanks a lot for your quick response.
